src/NVelocity.Tests/Bugs/NVelocity14.cs

```diff
@@ -35,17 +35,8 @@ public void Test()
 			extendedProperties.SetProperty(RuntimeConstants.FILE_RESOURCE_LOADER_PATH, TemplateTest.FILE_RESOURCE_LOADER_PATH);
 
 			velocityEngine.Init(extendedProperties);
-
-			VelocityContext context = new VelocityContext();
-
-			Template template = velocityEngine.GetTemplate(
-				GetFileName(null, "nv14", TemplateTest.TMPL_FILE_EXT));
-
-			StringWriter writer = new StringWriter();
-
-			template.Merge(context, writer);
-
-			Console.WriteLine(writer);
+			
+			Assert.Throws<ParseErrorException>(() => velocityEngine.GetTemplate(GetFileName(null, "nv14", TemplateTest.TMPL_FILE_EXT)));
```

Comment:
When converting to NUnit 3, I had to rewrite the test because of the removal of `ExpectedException`

The test actually fails with the expected exception when calling GetTemplate, so everything under that was never run. I just removed it since it didn't seem necessary.